### ISSUE TITLE
Don't display "Copy All Content" in new, empty, posts

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#15500]
When creating a new post and the post doesn't have a title or some content yet, the "Copy All Content" button doesn't do anything:

<img width="1279" alt="copy all content new empty post" src="https://user-images.githubusercontent.com/1682452/57376138-8d8bd100-719f-11e9-9e1a-9afc2ab9f484.png">

To reproduce:
- create a new post
- make sure to not enter any title or content
- open the toolbar "More" menu
- click on "Copy All Content"
- nothing happens

Instead, when there's some content, clicking the "Copy All Content" ... copies the content, and the button text changes to "Copied!".

Note: filling in only the title post doesn't seem to enable the copy functionality.

As this button doesn't do anything without content, it shouldn't be displayed until there's actually something to copy.

Accessibility notes:
- the button text shouldn't change dynamically: button text gives the button its accessible name which should inform users about _what_ the button does. It shouldn't be used for feedback or to indicate the _state_ of an action/setting. The "Copied!" text should appear separately.
- to evaluate: use an audible `speak()` message to confirm the copy operation result

**Comment by Soean** [comment]
In #16265 I added a snackbar notice, so we can remove the text change. I opened a PR: #16285

In a new empty post, should we disable the button or hide it?

Related: We are not sure if we can close the menu after the content ist copied: #16279

**Comment by afercia** [comment]
I've commented on https://github.com/WordPress/gutenberg/pull/16285#issuecomment-505516172

Please keep the text change for now. The "snackbars" implementation still needs to be evaluated by the accessibility team and I foresee a few problems to solve.

The readme itself states that when "snackbars" _contain an optional action..._
> It can also offer the user options, like viewing a published post but **these options should also be available elsewhere in the UI**.

For accessibility, this principle applies also to "snackbars" _without_ actions:
- the "snackbar" appears at the bottom left of the screen: this placement is out of view for users with limited vision field, users who use screen magnification, or zoom at high levels
- it goes against the principle of [proximity of controls](https://core.trac.wordpress.org/ticket/40822)
- it disappears after 10 seconds, violating the WCAG requirement related to time-based operation, see https://www.w3.org/TR/WCAG21/#enough-time
